src/lib/irap_export_ascii.cpp

```diff
@@ -0,0 +1,68 @@
+#include "include/irap_export.h"
+#include <cmath>
+#include <fstream>
+#include <iomanip>
+#include <ostream>
+#include <sstream>
+
+void write_header_ascii(const irap_header& header, std::ostream& out) {
+  out << std::setprecision(6) << std::fixed << std::showpoint;
+  out << "-996 " << header.ny << " " << header.xinc << " " << header.yinc
```

Comment:
-996 some kind of magic value?

Reply:
It is. I can make it a constant variable with a comment.